src/codespaces/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'codespaces' Extension
+==========================================
+
+This package is for the 'codespaces' extension.
+i.e. 'az codespaces'
```

Comment:
Can you add more introduction to this extension or a link to the service documentation?

Reply:
Added link to information on the service.